### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by KevinRoebert** [anonaddy/anonaddy#125] [Feature Requests]
Hello,

I would suggest localization of AnonAddy. Laravel even provides a [feature](https://laravel.com/docs/8.x/localization) for this and the standard messages are already available for [download](https://github.com/Laravel-Lang/lang) in many languages.

I tried it myself once [here](https://github.com/KevinRoebert/anonaddy/commit/88fb547c886c1758e392f2b2ae44f1dc70d72835), but it's been too long since I programmed something in Laravel and I think it would be easier if you include the feature, since you will know every place where changes have to be made.

I would be willing to do the complete translation to German as soon as the Laravel localization is integrated.

I can also recommend a translation management system like POEditor (as an open-source developer), where even not so technical people can contribute to the translation. I use this myself for my project ([ClearURLs](https://github.com/ClearURLs)) and it is very well accepted.

Regards,
Kevin

**Comment by HeikoBornholdt** [comment]
Another German user here. I would also be happy about this feature and assist with the translation.

**Comment by willbrowningme** [comment]
Thanks, that is extremely generous of you. Yes I am aware of the Laravel localization feature although I've not used it myself yet.

The front end pages for the web app are mainly Vue components so I would need to figure out how to get that to work nicely with Laravel localization.

I'm considering adding [Inertia.js](https://inertiajs.com/) at some point soon too, as I think it will make building the frontend easier and give a better "SPA" feel.

I've added this to the [roadmap](https://github.com/anonaddy/anonaddy/projects/1#card-54312549) and will do some more research into the best way to implement it.

**Comment by KevinRoebert** [comment]
@willbrowningme Any update?

**Comment by willbrowningme** [comment]
@KevinRoebert not at the moment I'm afraid. I'm currently working on upgrading the web app to Vue 3 and then the entire user interface. Hopefully once I've done that I can look at getting the site localization ready.

**Comment by pha099** [comment]
@willbrowningme  
Hi,Will.
Any update?

**Reply by willbrowningme** [reply]
No updates at the moment I'm afraid. I've been very busy with the rebrand and still have a long to do list to get through.

**Reply by pha099** [reply]
Any update?

**Comment by KevinRoebert** [comment]
@willbrowningme Hi Willi, any update? I'm still willing to help translate AnonAddy :)
